crates/net/network/src/eth_requests.rs

```diff
@@ -98,7 +98,7 @@ where
         };
 
         let skip = skip as u64;
-        let mut total_bytes = APPROX_HEADER_SIZE;
+        let mut total_bytes = 0;
```

Comment:
> In practice, this will allow an additional item in the response. This shouldn't make a big difference.

yeah this will now allow 1 item over the soft limit
given that we only do approximations here (atm) this doesn't really matter anyway